pkg/apis/core/v1alpha1/federatedtypeconfig_types.go

```diff
@@ -78,7 +79,17 @@ type FederatedTypeConfigStatus struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// FederatedTypeConfig
+// FederatedTypeConfig programs federation to know about a single API resource.
```

Comment:
Would it make sense to document what the 'target' type is?

Reply:
Yeah...